src/shims/vue.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.vue' {
```

Comment:
did you intend to check this file in?

Reply:
I've added it so TSC can recognize Vue component as a module:
https://github.com/symbol/desktop-wallet/blob/04f9ed9477cadba3e0478ccff025b886e5a7fcbc/__tests__/views/modals/ModalTransactionCosignature.spec.ts#L23